cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -11,7 +11,6 @@ package api
 type KeyHandle struct {
 	Key     []byte `json:"key,omitempty"`
```

Comment:
you may have to define encoding if it is keybytes

Reply:
Added a comment to clarify that it's raw key bytes.